<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFE_Window</class>
 <widget class="QMainWindow" name="QFE_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>832</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Colorlab FFCommand Engine</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>841</width>
      <height>761</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>FFmpeg</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>811</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Input File Commands</string>
      </property>
      <widget class="QFETableWidget" name="file_table">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>651</width>
         <height>121</height>
        </rect>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragDrop</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::MoveAction</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>70</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>File Dlg</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Parameter</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>File Name</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="add_input_btn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>121</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Arg</string>
       </property>
      </widget>
      <widget class="QPushButton" name="del_input_btn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>121</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete Arg</string>
       </property>
      </widget>
      <widget class="QPushButton" name="add_inputfiles_btn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>121</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Files Dialog</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="batch_cb">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Batch Mode</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>811</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Global Commands</string>
      </property>
      <widget class="QPushButton" name="del_param_btn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>114</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete Arg</string>
       </property>
      </widget>
      <widget class="QPushButton" name="add_param_btn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>114</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Arg</string>
       </property>
      </widget>
      <widget class="QFETableWidget" name="param_table">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>651</width>
         <height>81</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Parameter</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Value</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>811</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>Per Output Commands</string>
      </property>
      <widget class="QTabWidget" name="outputs_tw">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>771</width>
         <height>221</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>361</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QPushButton" name="delete_output_btn">
          <property name="text">
           <string>Delete Output</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="add_output_btn">
          <property name="text">
           <string>Add Output</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="Overwrite_cb">
          <property name="text">
           <string>Overwrite</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTextBrowser" name="ffcommand_tb">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>600</y>
        <width>741</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>660</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bauhaus 93</family>
        <pointsize>51</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel { 
border-image: url(:/colorlab.png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>670</y>
        <width>341</width>
        <height>33</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QSpinBox" name="maxinstance_sb">
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Max FFmpeg Instances</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="execute_btn">
         <property name="text">
          <string>Add to queue</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>FFprobe</string>
     </attribute>
     <widget class="QTreeView" name="treeView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>911</width>
        <height>541</height>
       </rect>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::InternalMove</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
      <attribute name="headerHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="headerMinimumSectionSize">
       <number>225</number>
      </attribute>
     </widget>
     <widget class="QPushButton" name="probe_btn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Probe Input File</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="ProbeOutput_tb">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>831</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="ffparse_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Parse Command</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_btn">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_pset_btn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save to Presets</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showqueue_btn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Queue</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>863</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSet_FFmpeg_Dir"/>
    <addaction name="actionAbout_FFCommand_Engine"/>
   </widget>
   <widget class="QMenu" name="menuPresets">
    <property name="title">
     <string>Presets</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPresets"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSet_FFmpeg_Dir">
   <property name="text">
    <string>Set FFmpeg Dir...</string>
   </property>
  </action>
  <action name="actionAbout_FFCommand_Engine">
   <property name="text">
    <string>About FFCommand Engine</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QFETableWidget</class>
   <extends>QTableWidget</extends>
   <header>qfetablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
